packages/components/src/snackbar/index.tsx

```diff
@@ -60,6 +58,7 @@ function UnforwardedSnackbar(
 		onRemove,
 		icon = null,
 		explicitDismiss = false,
+		duration = 10000,
```

Comment:
We might want to ensure that this prop is also documented in the component's README.